### ISSUE TITLE
Bride always facing same side, cant rotate

### ISSUE THREAD

**Issue by ardissaps** [TerraformersMC/Campanion#171]

**Comment by Wyn-Price** [comment]
Can you provide a screenshot

**Comment by ardissaps** [comment]
> Can you provide a screenshot

No matter how I built always facing West East. The rope post.
https://ibb.co/10JHQQc

**Comment by NeusFear** [comment]
This is not a bug, the post has no way of knowing what direction to face when placed. Once you connect the posts by building the rest of the bridge the ends turn to face one another

**Comment by Wyn-Price** [comment]
You need to connect two posts together with rope for it to work

**Comment by ardissaps** [comment]
> You need to connect two posts together with rope for it to work



> This is not a bug, the post has no way of knowing what direction to face when placed. Once you connect the posts by building the rest of the bridge the ends turn to face one another

yes it faced properly when connected, just some minor aesthetic issue. maybe can be fixed. Just like vanilla door that face according to position we are facing and standing.

**Comment by Wyn-Price** [comment]
I can maybe make I when you place the poles down they orient in the direction you're facing

**Comment by ardissaps** [comment]
> I can maybe make I when you place the poles down they orient in the direction you're facing

Yes please, thank you sir. :p

**Comment by ardissaps** [comment]
in addition please consider my suggestion :D

[Please support trinket slot. #170
](https://github.com/TerraformersMC/Campanion/issues/170)
